src/modules/account/components/BackupRecoveryPhraseFlow/BackupRecoveryPhraseFlow.js

```diff
@@ -12,7 +12,7 @@ const BackupRecoveryPhraseFlow = () => {
   const [passphrase, setPassphrase] = useState('');
 
   const onEnterPasswordSuccess = ({ recoveryPhrase }) => {
-    setPassphrase(recoveryPhrase);
+    setPassphrase(recoveryPhrase || '');
```

Comment:
shouldn't we just don't call the fn if !recoveryPhrase ?

Reply:
`if (recoveryPhrase) setPassphrase(recoveryPhrase)` might be a good option in this case since passphrase is `''` by default.